web/core/src/jsMain/kotlin/androidx/compose/web/elements/Base.kt

```diff
@@ -85,3 +185,14 @@ fun <TTag : Tag, TElement : Element> TagElement(
         refEffect.effect?.invoke(this, scope.element) ?: onDispose {}
     }
 }
+
+@Composable
+fun <TTag : Tag, TElement : Element> TagElement(
```

Comment:
If there is a variant of this function with `elementBuilder: ElementBuilder<TElement>`, then what's the need for this function with `tagName: String`? 

It's not used anywhere so far, right?

Reply:
I can imagine situation where people are already using this for creating custom tags.